MonoGame.Framework/Windows8/ModernHelper.cs

```diff
@@ -1,14 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
+﻿// MIT License - Copyright (C) The Mono.Xna Team
```

Comment:
This for sure should have a MonoGame license header...  I wrote this code myself for MonoGame.